src/autocomplete/StreamAutocomplete.js

```diff
@@ -14,11 +14,13 @@ class StreamAutocomplete extends Component {
   render() {
     const { filter, subscriptions, onAutocomplete } = this.props;
     const streams = subscriptions
-      .filter(x => x.name.toLowerCase().startsWith(filter.toLowerCase()))
-      .slice(0, 5);
```

Comment:
Good thinking.
We are using FlatList instead of ListView.
Let's try with that.